lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -41,12 +53,15 @@ While running the module, the application cannot do the following:
 
 Additionally, you must apply workarounds for the following limitations:
 
-* TLS through the modem is available, but requires that the application respects the LWM2M_CARRIER_EVENT_FOTA_START event.
+* TLS through the modem is available, but requires that the application respects the ``LWM2M_CARRIER_EVENT_FOTA_START`` event.
 * TLS handshakes through the modem might fail if the module is performing one at the same time.
   Implement a retry mechanism so that the application can perform its handshake later.
 
-See the changelog for the latest updates in the library, and for the list of known issues and limitations.
-The changelog also contains information about the certification status of each library.
+See the changelog for the latest updates in the library, and for a list of known issues and limitations.
+
+Certification
+*************
+The changelog contains information about the certification status of each version of the library.
 Your final product will typically need certification from the carrier as well.
 Contact the carrier for information about their device certification program.
 
```

Comment:
The changelog appears as subtopic to certification - intended?

Reply:
no, fixed